src/lib/components/table/_index.scss

```diff
@@ -212,6 +212,16 @@
   }
 }
 
+.vuiTableHeaderCell {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.vuiTableHeaderCell__label {
+  margin-right: 4px;
```

Comment:
How about using `gap` on `vuiTableHeaderCell` instead? One fewer class name to worry about, and this way it will only apply the spacing when the icon is visible.